iceoryx_examples/ice_multi_publisher/README.md

```diff
@@ -0,0 +1,262 @@
+# Multiple publishers for the same topic
+
+## Introduction
+A common use case is that often we have multiple sensors sending data of the same type, e.g. LIDAR data, and a subscriber is interested in the data of all those sensors. 
+
+This example shows how you can run multiple publishers and let them pblish of the same topic. 
+Here we only cover the specifics for this use case, for basic sending and receiving see the 
+ [icedelivery](../icedelivery/README.md) example. The example also demonstrates how you can change the maximum number of samples available at a subscriber when you resubscribe.
```

Comment:
maximum number of samples that can be queued up, I would say. How many are really available depends also on othere aspects like how many are published